app/tailwind.css

```diff
@@ -1,152 +1,109 @@
 
+/* TODO: remove this when react daisy component is integrated */
 @theme {
-  --font-sans:
-    Inter, ui-sans-serif, system-ui, sans-serif, Apple Color Emoji,
-    Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
-
-  --radius-lg: var(--radius);
-  --radius-md: calc(var(--radius) - 2px);
-  --radius-sm: calc(var(--radius) - 4px);
-
   --color-azure-10: #131720;
   --color-azure-15: #1f242f;
   --color-azure-20: #0c111d;
   --color-azure-25: #12192a;
 
-  --color-gray-10: #6d717b;
-
-  --color-white-3: rgba(255, 255, 255, 0.03);
-  --color-dark-blue: #060b1a;
-  --color-white-10: rgba(255, 255, 255, 0.1);
   --color-background: hsl(var(--background));
   --color-foreground: hsl(var(--foreground));
-
-  --color-card: hsl(var(--card));
-  --color-card-foreground: hsl(var(--card-foreground));
-
-  --color-popover: hsl(var(--popover));
-  --color-popover-foreground: hsl(var(--popover-foreground));
-
-  --color-primary: var(--primary);
   --color-primary-foreground: var(--primary-foreground);
-
-  --color-secondary: hsl(var(--secondary));
-  --color-secondary-foreground: hsl(var(--secondary-foreground));
-
   --color-muted: hsl(var(--muted));
-  --color-muted-foreground: hsl(var(--muted-foreground));
-
-  --color-accent: hsl(var(--accent));
-  --color-accent-foreground: hsl(var(--accent-foreground));
 
-  --color-destructive: var(--destructive);
-  --color-destructive-foreground: var(--destructive-foreground);
-
-  --color-border: hsl(var(--border));
-  --color-input: hsl(var(--input));
-  --color-ring: hsl(var(--ring));
-
-  --color-chart-1: hsl(var(--chart-1));
-  --color-chart-2: hsl(var(--chart-2));
-  --color-chart-3: hsl(var(--chart-3));
-  --color-chart-4: hsl(var(--chart-4));
-  --color-chart-5: hsl(var(--chart-5));
+  --foreground: 0 0% 98%;
+  --popover: 240 10% 3.9%;
+  --popover-foreground: 0 0% 98%;
+  --primary-foreground: #ee842c;
+  --secondary: rgba(40, 40, 40, 1);
+  --secondary-foreground: rgba(250, 250, 250, 1);
+  --muted: rgba(40, 40, 40, 1);
+  --accent: rgba(40, 40, 40, 1);
+  --destructive: #d00416;
+  --destructive-foreground: #fb3748;
+  --ring: 240 4.9% 83.9%;
 }
 
 /*
-  The default border color has changed to `currentcolor` in Tailwind CSS v4,
-  so we've added these compatibility styles to make sure everything still
-  looks the same as it did with Tailwind CSS v3.
+  This is the main daisyUI plugin configuration.
+  It enables two custom themes: 'native-light' and 'native-dark'.
+  - 'native-light' is the default theme for the site.
+  - 'native-dark' is automatically applied if the user prefers a dark color scheme.
+*/
+@plugin "daisyui" {
+  themes: dark --default;
+  logs: false; /* Disables daisyUI console logs for a cleaner console */
+}
 
-  If we ever want to remove these styles, we need to add an explicit border
-  color utility to any element that depends on these defaults.
+/*
+  ==========================================================================
+  NATIVE LIGHT THEME
```

Comment:
We have only one theme. update the name, because below it's `name: "dark"`

Reply:
Done